### PR TITLE
feat(css): Add the `::‑webkit‑scrollbar` selectors

### PR DESCRIPTION
**Pull request by ExE-Boss**
I have confirmed that this works on iOS.

review?(@ddbeck, @Elchi3)

### CODE REVIEW

File: css/selectors/-webkit-resizer.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "deprecated": false,
+            "experimental": false,
+            "standard_track": false
```
Comment (ddbeck) [inline css/selectors/-webkit-resizer.json]: Can these status blocks be sorted in the usual experimental-standard-deprecated order? This applies to each of the files in this PR.

Comment (ExE-Boss) [reply css/selectors/-webkit-resizer.json]: Done.

Also, it turns out that a lot of compat data uses the alphabetical order for&nbsp;the&nbsp;**status** block.

**Comment by ExE-Boss** [comment]
review?(@ddbeck): I have now addressed your previous review.

**Comment by ddbeck** [comment]
Thank you, @ExE-Boss! 🎉

**Pull request merged**